spec/unit/full_transmission_spec.rb

```diff
@@ -70,7 +70,7 @@
   # provides: input_events
   # depends: output_events
   shared_examples "large sequence" do
-    let(:event_count) { 10_000 }
+    let(:event_count) { 10_0 }
```

Comment:
review note: intentionally reduced the count because CIs are failing due too many logs. More logs are generated because for the failure cases, we continuously retry sending events over hosts, before we had a sleep with http-output.